rules/game_process.tex

```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands, covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}, straight over its head.
+The GameController signal for the transition from \texttt{initial} to \texttt{ready} will be delayed by \qty{\SetDelayTimeChampion}{\second} for champions cup and by \qty{\SetDelayTimeChallenge}{\second} for challenge shield. 
+The referee must stand on the opposite side of the game controller behind the touchline and at the height of the halfway line. 
```

Comment:
This definition probably belongs earlier to understand the context of the signals. See my other comment above.